manifests/config.pp

```diff
@@ -110,14 +110,16 @@
     subscribe => Exec['Directory permissions'],
   }
 
-  exec{'setsebool redis':
-    command   => '/usr/sbin/setsebool -P httpd_can_network_connect on',
-    unless    => '/usr/sbin/getsebool httpd_can_network_connect | grep -e  "--> on"',
-    subscribe => File['/etc/opt/rh/rh-php56/php.d/99-redis.ini'],
+  if $facts['os']['selinux']['enabled'] {
```

Comment:
the ```$facts['os']['selinux']``` hash is not always present :(. Please use [fact()](https://forge.puppet.com/puppetlabs/stdlib#fact) instead. This requires at least stdlib 4.19. Please bump the lowest supported version in our [metadata.json](https://github.com/voxpupuli/puppet-misp/blob/master/metadata.json#L19).